presto-product-tests/README.md

```diff
@@ -43,6 +43,10 @@ broken.
 `hadoop-master`, `mysql`, `postgres`, `cassandra`, `presto-master`.
 They should point to your external IP address (shown by `ifconfig` on your Mac (not inside docker)).
 
+* The default memory setting of 2GB might not be sufficient for some profiles like singlenode-ldap.
+To increase docker memory, go to Docker Preferences > Advanced > Memory, and increase it appropriately.
+We recommend at least 6GB for such profiles.
```

Comment:
Why don't you rearrange it like this:
``The default memory setting of 2GB might not be sufficient for some profiles like singlenode-ldap. We recommend at least 6GB for such profiles. You can increase docker memory by following Docker Preferences -> Advanced -> Memory.
 ``